### PR TITLE
Feature/uwp attachments

### PR DESCRIPTION
**Pull request by michaelsmitha**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Add text and binary attachments to uwp apps

## Related PRs or issues

[AB#70775](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/70775)

### CODE REVIEW

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/FilePicker.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(FilePicker))]
+namespace Contoso.Forms.Puppet.UWP
+{
+    public class FilePicker : IFilePicker
+    {
+
```
Comment (guperrot) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/FilePicker.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml [hunk 2]
```diff
@@ -4,8 +4,21 @@
         <TableSection Title="Crashes Settings">
             <SwitchCell Text="Crashes Enabled" On="true" x:Name="CrashesEnabledSwitchCell" OnChanged="UpdateEnabled" OnColor="{StaticResource SwitchCellOnColor}"  />
             <SwitchCell Text="Handle Exceptions" On="false" x:Name="HandleExceptionsSwitchCell" OnColor="{StaticResource SwitchCellOnColor}" />
-            <TextCell Text="Text Attachment" x:Name="TextAttachmentCell" Tapped="TextAttachment" />
-            <TextCell Text="Binary Attachment" x:Name="FileAttachmentCell" Tapped="FileAttachment" />
+            <ViewCell Tapped="TextAttachment">
```
Comment (guperrot) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml]: Was that tested on Xamarin? This is shared code.

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs [hunk 3]
```diff
@@ -52,27 +52,22 @@ protected override async void OnAppearing()
             if (Application.Current.Properties.TryGetValue(TextAttachmentKey, out var textAttachment) &&
```
Comment (guperrot) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs]: Was that tested on Xamarin? This is shared code.

File: Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/FilePicker.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Xamarin.Forms;
+
+[assembly: Dependency(typeof(FilePicker))]
+namespace Contoso.Forms.Puppet.UWP
+{
+    public class FilePicker : IFilePicker
+    {
+
```
Comment (guperrot) [inline Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/FilePicker.cs]:
```suggestion
```

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml (hunk 2 above)
Comment (dhei) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml]: Tested with Xamarin.Forms Puppet iOS and Demo Android.

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs (hunk 3 above)
Comment (dhei) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs]: Tested with Xamarin.Forms Puppet iOS and Demo Android.

**Pull request merged**